tools/license2rtf.js

```diff
@@ -1,15 +1,16 @@
+'use strict';
 
 var assert = require('assert'),
-    Stream = require('stream'),
-    inherits = require('util').inherits;
+  Stream = require('stream'),
+  inherits = require('util').inherits;
```

Comment:
I agree. We changed a ton of these multiline declarations a while ago, so it would be more consisten to also have seperate declarations here.